### ISSUE TITLE
Apply flags/extensions and restart button in the session section does not work properly

### ISSUE THREAD

**Issue by Touch-Night** [oobabooga/text-generation-webui#3839]
### Describe the bug

The button can only enable the flags/extensions selected and restart the webui, can't disable unselected extensions.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Enable an extension for example google translate, apply & restart, and then uncheck it, apply & restart, the google translate part is still visible in chat section and is usable.

### Logs

```shell
No error in command-line
```

### System Info

```shell
System: Windows 10
GPU: Nvidia RTX 3050Ti Laptop (modified to 8GB RAM)
```

**Comment by Touch-Night** [comment]
Extensions loaded [here](https://github.com/oobabooga/text-generation-webui/blob/4affa088214b59a8bbefbd0f956523a396486bd7/modules/extensions.py#L35C3-L35C3) with `import` never gets deleted after `shared.gradio['interface'].close()`, resulting in unexpected load [here](https://github.com/oobabooga/text-generation-webui/blob/4affa088214b59a8bbefbd0f956523a396486bd7/server.py#L135-L136)

**Comment by Touch-Night** [comment]
Fixed in #3827

**Comment by Touch-Night** [comment]
Close as fixed
